Chessington.GameEngine/Pieces/Bishop.cs

```diff
@@ -10,7 +10,28 @@ public Bishop(Player player)
 
         public override IEnumerable<Square> GetAvailableMoves(Board board)
         {
-            return Enumerable.Empty<Square>();
+            List<Square> availableMoves = new List<Square>();
+            Square here = board.FindPiece(this);
```

Comment:
I feel like something like "currentPosition" will be a better name for this variable